content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -61,7 +61,12 @@ Recent key learnings:
 
 ### Competitive landscape
 
-- **Code hosts:** Both GitHub and GitLab have navigation features, mostly powered by imprecise code intelligence. They have good out-of-the-box support, but their chosen technical path prioritizes wider language breath sacrificing accuracy. Our precise code intelligence approach focuses on having 100% accuracy by implementing LSIF indexers. So far none of our competitors have mimicked the approach, but as precise code navigation gains traction this is a potential risk.
+- **Code hosts:** Both GitHub and GitLab offer navigation features, mostly powered by fuzzy code intelligence.
+
+  GitHub has recently released a new version of their [Code Search](https://cs.github.com/) which includes an enhanced version of our search-based code intelligence approach. This means their chosen technical path prioritizes wider language breath while sacrificing accuracy. However, if GitHub's Code Search navigation proves to be "good enough" for most use cases, we'll have to directly compete for adoption and usage.
```

Comment:
why the spaces lol

Reply:
Felt a bit long to be a single paragraph, happy to remove if they feel unnecessary though!